frinx-uniconfig/user-guide/network-management-protocols/uniconfig_netconf/readme.md

```diff
@@ -964,6 +965,90 @@ Notes:
 3. --initial-config-xml-file data.xml overrides --md-sal true and send hardcoded response to get-config.
 4. --notification-file notif.xml notification support
 
+### Starting of the NETCONF testtool from Docker Image
+
+1. download docker image using download command
+```
+docker pull frinx/netconf-testtool
+```
+2. run it using docker run command
+```
+docker run -it frinx/netconf-testtool
+```
+3. you can use options of Netconf test tool starting script
+```
+-x or --xmx to set Java heap for test tool
+-p or --port to set starting port
+-d or --devices to set number of simulated devices
+-s or --schemas to set schemas folder (required firstly mount schemas folder to docker container using -v)
+-i or --init-conf to set initial simulated config to be returned by get-config RPC (required firstly mount this config file to docker container using -v)
+--ssh to enable ssh
+--md-sal to enable md-sal
+--debug to run in debug mode
+-h or --help to display help options
+--override to override these options to access all netconf test tool options directly
+```
+**note**  
+To set value of xmx is required to enter xmx heap size value separated from -x or --xmx option.  
```

Comment:
i do not quite understand.
what does mean separated in the context of all sentence?
ahhh i remember
you mean there must be a space between keyword and value.
please rephrase e.g.
while using xmx option you must use space before entered value